jobs/test-override.Jenkinsfile

```diff
@@ -58,17 +58,29 @@ def blueocean_url = "${env.RUN_DISPLAY_URL}"
 
 def bodhi_update_ids = []
 def koji_build_ids = []
+def koji_scratch_build_ids = []
+def copr_build_ids = []
+
+def bodhi_url_prefix = "https://bodhi.fedoraproject.org/updates/"
+def koji_url_prefix = "https://koji.fedoraproject.org/koji/buildinfo?buildID="
+def koji_scratch_url_prefix = "https://koji.fedoraproject.org/koji/taskinfo?taskID="
+def copr_url_prefix = "https://copr.fedorainfracloud.org/coprs/g/CoreOS/continuous/build/"
```

Comment:
Is there any way here to allow builds from any COPR repo to be submitted and not just the CoreOS continuous repo?

For example. Koji scratch builds can be submitted for any build, not just specific ones. Not sure if there is a way to do that with COPR here or not.

Reply:
Good catch, if using `copr-cli`, then could download the build, but we still need the URL, for example https://copr.fedorainfracloud.org/coprs/g/rubygems/rubygems/build/8957089/
`copr-cli --debug download-build --chroot fedora-42-x86_64 8957089 --rpms`

Reply:
Maybe for copr build, we can just use `copr/8957089`? Then it would not limit CoreOS continuous project.

Reply:
yeah. I think that sounds good

Reply:
Thanks for the suggestion, updated.